### ISSUE TITLE
Feature request: Support for `comment` statement in PostgreSQL

### ISSUE THREAD

**Issue by shiroemons** [k0kubun/sqldef#266]
I would like to add comments to tables, columns, etc. in psqldef.

Could you please support [the `comment` statement](https://www.postgresql.org/docs/current/sql-comment.html)?

## --export output

```SQL
CREATE TABLE "public"."hoge" (
    "id" bigserial NOT NULL,
    "foo" text NOT NULL DEFAULT ''::text,
    PRIMARY KEY ("id")
);
```

## Input SQL

```SQL
CREATE TABLE "public"."hoge" (
    "id" bigserial NOT NULL,
    "foo" text NOT NULL DEFAULT ''::text,
    PRIMARY KEY ("id")
);

COMMENT ON TABLE hoge is 'hoge table';
COMMENT ON COLUMN hoge.id is 'hoge id';
COMMENT ON COLUMN hoge.foo is 'foo comment';
```

## Current output

```
found syntax error when parsing DDL "COMMENT ON TABLE hoge is 'hoge table';
COMMENT ON COLUMN hoge.id is 'hoge id';
COMMENT ON COLUMN hoge.foo is 'foo comment'": syntax error at position 8 near 'comment'
```

- psql
    ```SQL
    sample=# COMMENT ON TABLE hoge is 'hoge table';
    COMMENT
    sample=# COMMENT ON COLUMN hoge.id is 'hoge id';
    COMMENT
    sample=# COMMENT ON COLUMN hoge.foo is 'foo comment';
    COMMENT
    ```

## Expected output

**Comment by k0kubun** [comment]
Could you please follow the issue template? You need to provide an input SQL you actually need to support.

**Comment by shiroemons** [comment]
Updated Summary.

**Comment by k0kubun** [comment]
Thank you. I think this issue is ready to work on.

As I'm currently focusing on a big change for another OSS, I'm planning to work on this issue on Aug 13th. If I receive a pull request before that, I can take a look at that too.

**Comment by k0kubun** [comment]
Fixed in v0.13.2.
